airbyte-cdk/python/airbyte_cdk/base_python/integration.py

```diff
@@ -83,17 +83,19 @@ def spec(self, logger: AirbyteLogger) -> ConnectorSpecification:
         Returns the spec for this integration. The spec is a JSON-Schema object describing the required configurations (e.g: username and password)
         required to run this integration.
         """
-        raw_spec = pkgutil.get_data(self.__class__.__module__.split(".")[0], "spec.json")
+        raw_spec: Optional[bytes] = pkgutil.get_data(self.__class__.__module__.split(".")[0], "spec.json")
+        if not raw_spec:
+            raise ValueError("Unable to find spec.json.")
         return ConnectorSpecification.parse_obj(json.loads(raw_spec))
 
-    def check(self, logger: AirbyteLogger, config: json) -> AirbyteConnectionStatus:
+    def check(self, logger: AirbyteLogger, config: Mapping[str, Any]) -> AirbyteConnectionStatus:
```

Comment:
These function signatures previously did not match the classes implementing them. This now matches.

Reply:
👍🏼